### ISSUE TITLE
Make 34.0.5 release

### ISSUE THREAD

**Issue by anatol** [nmeum/android-tools#131]
Upstream android has finally tagged the source tree with 34.0.5 tag.

@Biswa96 when you have a chance please update the repo to the new upstream release.

**Comment by Biswa96** [comment]
I have tried that but got compiler error with clang in every distribution.

```
In file included from /__w/andy/andy/android-tools-34.0.5/vendor/adb/types.cpp:17:
/__w/andy/andy/android-tools-34.0.5/vendor/adb/types.h:235:13: error: static assertion failed due to requirement 'std::is_standard_layout<Block>()'
            static_assert(std::is_standard_layout<decltype(result)>());
            ^             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/__w/andy/andy/android-tools-34.0.5/vendor/adb/types.cpp:190:47: note: in instantiation of function template specialization 'IOVector::coalesce<Block>' requested here
    return const_cast<const IOVector*>(this)->coalesce<>();
                                              ^
1 error generated.
```

Also, all macos builds fail which needs some investigation.

If you allow I can create a draft pull request with the changes.

**Comment by anatol** [comment]
Yes please create a draft PR and me or someone else will look at the compiler error.
